galaxy_ng/app/migrations/0008_rename_default_repo.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.15 on 2020-09-17 16:55
+
+from django.db import migrations
+
+def rename_default_repo(apps, schema_editor):
```

Comment:
Minor: Usually it's preferable to write an SQL migration rather than a Python migration when possible.